accname/name_test_case_shadowdom_1-manual.html

```diff
@@ -0,0 +1,84 @@
+<!doctype html>
+<html>
+  <head>
+    <title>Name test case 762</title>
+    <meta content="text/html; charset=utf-8" http-equiv="Content-Type"/>
+    <link rel="stylesheet" href="/wai-aria/scripts/manual.css">
+    <style>
+      button {
+        width: 75px;
+        height: 25px;
+      }
+    </style>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <script src="/wai-aria/scripts/ATTAcomm.js"></script>
+    <script>
+    setup({explicit_timeout: true, explicit_done: true });
+
+    var theTest = new ATTAcomm(
+    {
+   "steps" : [
+      {
+         "element" : "test",
+         "test" : {
+            "ATK" : [
+               [
+                  "property",
+                  "name",
+                  "is",
+                  "foo"
+               ]
+            ],
+            "AXAPI" : [
+               [
+                  "property",
+                  "AXDescription",
```

Comment:
I think in this scenario, we should be checking `AXTitle`. Accname for the AXAPI is a little different than the rest -- the accname is exposed in `AXTitle` when visually rendered or `AXDescription` when not visually rendered: https://w3c.github.io/core-aam/#accessible-names-and-descriptions